app/models/miq_expression/field.rb

```diff
@@ -53,8 +63,58 @@ def does_not_match(other)
     arel_attribute.does_not_match(other, escape, case_sensitive)
   end
 
+  def contains(other)
+    raise unless associations.one?
+    reflection = reflections.first
+    arel = eq(other)
+    arel = arel.and(Arel::Nodes::SqlLiteral.new(extract_where_values(reflection.klass, reflection.scope))) if reflection.scope
+    model.arel_attribute(:id).in(
+      target.arel_table.where(arel).project(target.arel_table[reflection.foreign_key]).distinct
+    )
+  end
+
   private
 
+  class WhereExtractionVisitor < Arel::Visitors::PostgreSQL
```

Comment:
@kbrock Two things:

a)  That is a @imtayadeway question
b)  This is something that was already in place, and was just moved:  https://github.com/ManageIQ/manageiq/pull/13285/files#diff-ff5e4f0f23adaff0ec06a295cd2376f8L1921